docs/03-Cloud-Native-Journey/02-learning-goals.adoc

```diff
@@ -2,90 +2,118 @@
+
+Softwarearchitekt:innen kennen die verschiedenen Wege um Daten zu persistieren.
+
+Sie kennen den Unterschied von Object- und Block-Storage und können die verschiedenen Storage-Dienste der Cloud-Anbieter, unterschiedliche COTS-Produkte und selbstverwaltete Lösungen für unterschiedliche Einsatzszenarien klassifizieren.
+
+Softwarearchitekt:innen verstehen die Vor- und Nachteile einer Cloud-Native-Storage-Lösung und können die höherwertigen Managed Services (wie etwa RDBMS) der Cloud-Anbieter mit der Verwendung selbstverwalteter Lösungen vergleichen.
+
+Sie kennen verschiedene Konzepte, um Backup und Restore von persistenten Daten in der Cloud zu realisieren.
 // end::DE[]
 
 // tag::EN[]
 [[LG-3-1]]
-==== LG 3-1: Cloud Native Concept
-Understand the benefits of the Cloud Native concept, particularly through unified...:
+==== LG 3-1: Understand the Cloud Native Concept
 
-* Interfaces for administration and configuration
+Software architects understand the benefits and goals of the Cloud Native concept. They are familiar with the advantages of the concept, particularly through:
+
+* Unified interfaces for administration and configuration
 * Resource abstraction
 * Delivery artifacts
-* Isolation of the application, with respects to the host, etc.
+
 
 [[LG-3-2]]
-==== LG 3-2: Container Manager
-* Understand the abstraction concepts of current container managers and know patterns and best practices for how to apply non-functional requirements (NFR) fulfillment to them. For example:
-** Finite workloads such as (cron) jobs
-** Health Check and Self Healing
-** Scaling and Load Balancing
-** Placement
-* Know the principles of an overlay network (VLAN) for containers and container managers and understand how to isolate different services or systems. E.g., implement a distributed firewall to isolate:
-** different clients
-** test systems in the context of a CI/CD
+==== LG 3-2: Understand basic container concepts
 
-[[LG-3-3]]
-==== LG 3-3: Container
-Understand the Linux kernel isolation mechanisms that apply to containers (container security), including:
+Software architects understand the technical basics and the added value of containers and know the difference between conventional deployment and deployment with containers.
+
+They understand how the use of containers leads to a better decoupling of responsibilities and thus to a more effective organization.
+
+Software architects understand Linux kernel isolation mechanisms that apply to containers (container security), including:
 
+* Namespaces and cgroups
 * Capabilities
-* Privileged Containers
-* AppArmor
+
+They also understand the implications, usage scenarios, and threats of privileged containers.
+
+[[LG-3-3]]
+==== LG 3-3: Understand Container Manager Abstraction Concepts
+
+Software architects understand the abstraction concepts of current container managers and know patterns and best practices on how to apply non-functional requirements (NFR) fulfillment to them. For example:
```

Comment:
`and know patterns and best practices on how to apply non-functional requirements (NFR) fulfillment to them.`

Das klingt irgendwie.. holprig. Vielleicht könnt ihr das nochmal besser schreiben?

Reply:
✔️